packages/expo-ads-admob/ios/EXAdsAdMob/EXAdsAdMobRewarded.m

```diff
@@ -85,6 +86,16 @@ - (void)stopObserving {
       extras.additionalParameters = additionalRequestParams;
       [request registerAdNetworkExtras:extras];
     }
+    if(ssvOptions) {
+      GADServerSideVerificationOptions *options = [[GADServerSideVerificationOptions alloc] init];
+      if(ssvOptions[@"userId"]) {
+        options.userIdentifier = ssvOptions[@"userId"];
+      }
+      if(ssvOptions[@"customData"]) {
+        options.customRewardString = ssvOptions[@"customData"];
+      }
```

Comment:
do we need to check and ensure each of these values that were provided by the user are a String? (assume not all users are using TS)

Reply:
Ok, I've added the explicit checks for [... isKindOfClass:[NSString class]].